### PR TITLE
Allow analytics to be disabled

### PR DESCRIPTION
**Pull request by davidrapson**
Models all analytics in JS, rather than html includes/script tags.
Allows analytics to be disabled, will only run if:

- `guardian.analyticsEnabled` is `true`
- `Navigator.doNotTrack` is not set
- We don't have an `ANALYTICS_OFF` cookie for test users

If the CSP test is successful on Membership we'll also be implementing this here, which will help to track the impact of these analytics.

@rtyley

### CODE REVIEW

File: assets/javascripts/modules/analytics/affectv.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+/*global Raven */
```
Comment (rtyley) [inline assets/javascripts/modules/analytics/affectv.js]: 'Raven'?

File: assets/javascripts/modules/analytics/krux.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/*global Raven */
```
Comment (rtyley) [inline assets/javascripts/modules/analytics/krux.js]: I'm guessing this C&P?

File: assets/javascripts/modules/analytics/affectv.js (hunk 1 above)
Comment (davidrapson) [inline assets/javascripts/modules/analytics/affectv.js]: It's the Sentry client; It's exposed as a global.

File: assets/javascripts/modules/analytics/krux.js (hunk 2 above)
Comment (davidrapson) [inline assets/javascripts/modules/analytics/krux.js]: As above

File: assets/javascripts/modules/analytics/affectv.js (hunk 1 above)
Comment (rtyley) [inline assets/javascripts/modules/analytics/affectv.js]: Oh, fair enough.

**Comment by rtyley** [comment]
:+1:

**Pull request merged**